### PR TITLE
Docs folder standardization and other fixes

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: resources/views/layout/main.php [hunk 1]
```diff
@@ -29,101 +29,81 @@
 $this->beginPage()
 ?><!DOCTYPE html>
 <html lang="<?= Html::encode($locale->language()) ?>">
+
```
Comment (samdark) [inline resources/views/layout/main.php:32]: Why?

File: resources/views/layout/main.php [hunk 2]
```diff
@@ -29,101 +29,81 @@
...
-            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
-                <path
-                    d="M10 0c5.522847 0 10 4.477152 10 10 0 5.522847-4.477153 10-10 10-5.522848 0-10-4.477153-10-10C0 4.477152 4.477152 0 10 0Zm4.44152 6c-.380967.00676-.965465.207411-3.778372 1.362498-.985186.404555-2.954171 1.241881-5.906955 2.51198-.479486.188247-.730662.372408-.753528.55248-.0439.345722.460248.45335 1.094572.656918.517165.16597 1.212843.360136 1.574497.36785.328055.006997.694202-.126527 1.098441-.400574 2.758878-1.838605 4.183023-2.767925 4.272434-2.78796.063079-.014134.15049-.031906.209715.020065.059226.05197.053403.150393.04713.176797-.050161.21112-2.641806 2.538403-2.790957 2.691345l-.071523.071907c-.54956.542806-1.105014.8972-.146641 1.52071.865332.56298 1.36896.922122 2.260436 1.499054.569705.368694 1.01636.805908 1.604566.75247.270659-.02459.550232-.275851.692217-1.025227.335553-1.771.995126-5.608216 1.147558-7.189441.013355-.138536-.003445-.315834-.016937-.393664-.013492-.07783-.041685-.188721-.144157-.27081-.121356-.097219-.308705-.11772-.392495-.116398Z"
-                />
+            <svg xmlns="https://www.w3.org/2000/svg" viewBox="0 0 1000 216.495">
```
Comment (samdark) [inline resources/views/layout/main.php:46]: xmlns should be HTTP, not HTTPs

**Review by samdark — CHANGES_REQUESTED** [review]

File: resources/views/layout/main.php (hunk 1 above)
Comment (luizcmarin) [reply resources/views/layout/main.php:32]: What?
To be
     "$this->beginPage()
     ?><!DOCTYPE html>"
and not
     "$this->beginPage()
     ?>
     <!DOCTYPE html>"
?

https://t.me/yii3en/9570

Should I change?

File: docs/internals.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+## Mutation testing
+
+The package tests are checked with [Infection](https://infection.github.io/) mutation framework with
+[Infection Static Analysis Plugin](https://github.com/Roave/infection-static-analysis-plugin). To run it:
+
+```shell
+./vendor/bin/roave-infection-static-analysis-plugin
+```
```
Comment (samdark) [inline docs/internals.md:18]: This package is tested with Codeception instead: https://github.com/yiisoft/app/tree/master/.github/workflows

File: resources/views/layout/main.php [hunk 4]
```diff
@@ -29,101 +29,81 @@
...
+
+    <div class="footer">
+        <div class="footer_copyright">
+            <a href="https://www.yiiframework.com/" target="_blank" rel="noopener">
+                © <?= date('Y') ?> <?= Html::encode($applicationParameters->getName()) ?>
+            </a>
+        </div>
+        <div class="footer_icons">
+            <a href="https://github.com/yiisoft" target="_blank" rel="noopener" title="GitHub">
+                <svg xmlns="https://www.w3.org/2000/svg" viewBox="0 0 291.32 291.32">
```
Comment (samdark) [inline resources/views/layout/main.php:78]:
```suggestion
                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 291.32 291.32">
```

**Review by samdark — CHANGES_REQUESTED** [review]

File: LICENSE.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
-Copyright © 2008 by Yii Software (https://www.yiiframework.com/)
+Copyright © 2008 by Yii Software (<https://www.yiiframework.com/>)
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
 are met:
 
- * Redistributions of source code must retain the above copyright
+* Redistributions of source code must retain the above copyright
```
Comment (vjik) [inline LICENSE.md:8]: Align it vertically, like here: https://raw.githubusercontent.com/yiisoft/package-template/master/LICENSE.md

File: README.md [hunk 6]
```diff
@@ -200,24 +198,25 @@ See ["Logging"](https://github.com/yiisoft/docs/blob/master/guide/en/runtime/log
...
+## Documentation
 
-The code is statically analyzed with [Psalm](https://psalm.dev/). To run static analysis:
+- [Internals](docs/internals.md)
 
-```shell
-./vendor/bin/psalm
-```
+## Support
+
```
Comment (vjik) [inline README.md:211]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -227,7 +226,7 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
 
 ## License
 
-The Yii application template is free software. It is released under the terms of the BSD License.
+The Yii Application Template for Yii 3 is free software. It is released under the terms of the BSD License.
```
Comment (vjik) [inline README.md:229]:
```suggestion
The Yii Application Template is free software. It is released under the terms of the BSD License.
```

File: resources/views/layout/main.php (hunk 1 above)
Comment (vjik) [inline resources/views/layout/main.php:32]:
```suggestion
```

File: resources/views/layout/main.php [hunk 8]
```diff
@@ -29,101 +29,81 @@
...
-                <path
-                    d="M-120.995-431.898c-5.761-16.855-3.32-28.149 7.22-43.866 5.027-7.499 13.706-16.197 21.313-21.111 30.691 19.232 42.039 55.054 30.752 88.323-8.214 24.207-15.921 34.357-35.408 59.263 2.271-26.674-7.117-45.167-16.334-65.715-2.346-5.228-5.59-11.179-7.543-16.894"
-                    style="fill-rule:evenodd;clip-rule:evenodd;fill:#f18a2a" transform="translate(233.564 496.875)"/>
-                <path
-                    d="M-115.623-336.617c.215-6.016-4.07-11.074-8.349-14.917-9.513-8.546-25.864-14.245-25.864-14.245 1.682-2.738 3.349-6.138 4.604-8.861 7.519-16.305 11.701-38.841 11.71-38.871.003-.01 4.657 6.62 7.685 13.703 3.029 7.085 9.248 19.584 10.71 38.869.425 5.625.859 10.005-.496 24.322z"
-                    style="fill:#7fb93c" transform="translate(245.403 498.558)"/>
-            </g>
-        </svg>
-    </a>
-</div>
```
Comment (vjik) [inline resources/views/layout/main.php:40]:
```suggestion
```

File: resources/views/layout/main.php [hunk 9]
```diff
@@ -29,101 +29,81 @@
...
+                    <path d="M10 0c5.522847 0 10 4.477152 10 10 0 5.522847-4.477153 10-10 10-5.522848 0-10-4.477153-10-10C0 4.477152 4.477152 0 10 0Zm4.44152 6c-.380967.00676-.965465.207411-3.778372 1.362498-.985186.404555-2.954171 1.241881-5.906955 2.51198-.479486.188247-.730662.372408-.753528.55248-.0439.345722.460248.45335 1.094572.656918.517165.16597 1.212843.360136 1.574497.36785.328055.006997.694202-.126527 1.098441-.400574 2.758878-1.838605 4.183023-2.767925 4.272434-2.78796.063079-.014134.15049-.031906.209715.020065.059226.05197.053403.150393.04713.176797-.050161.21112-2.641806 2.538403-2.790957 2.691345l-.071523.071907c-.54956.542806-1.105014.8972-.146641 1.52071.865332.56298 1.36896.922122 2.260436 1.499054.569705.368694 1.01636.805908 1.604566.75247.270659-.02459.550232-.275851.692217-1.025227.335553-1.771.995126-5.608216 1.147558-7.189441.013355-.138536-.003445-.315834-.016937-.393664-.013492-.07783-.041685-.188721-.144157-.27081-.121356-.097219-.308705-.11772-.392495-.116398Z" />
+                </svg>
+            </a>
+        </div>
+    </div>
+
+    <?php $this->endBody() ?>
 </body>
+
```
Comment (vjik) [inline resources/views/layout/main.php:107]:
```suggestion
```

**Review by vjik — CHANGES_REQUESTED** [review]

File: resources/views/layout/main.php [hunk 10]
```diff
@@ -37,93 +37,71 @@
...
+                </svg>
+            </a>
+            <a href="https://join.slack.com/t/yii/shared_invite/enQtMzQ4MDExMDcyNTk2LTc0NDQ2ZTZhNjkzZDgwYjE4YjZlNGQxZjFmZDBjZTU3NjViMDE4ZTMxNDRkZjVlNmM1ZTA1ODVmZGUwY2U3NDA" target="_blank" rel="noopener" title="Slack">
```
Comment (samdark) [inline resources/views/layout/main.php:80]: Let's link here: https://www.yiiframework.com/chat

**Pull request closed without merging**